database/dynamicseed.js

```diff
@@ -72,27 +72,27 @@ const generateExperienceEntry = function (index) { // eslint-disable-line
   return entry;
 };
 const insertSampleExperience = function (entry) { // eslint-disable-line
-   return experienceModel.create(entry)
-   .catch((error) => {
-     console.log('catch');
-     return error;
-   });
+  return experienceModel.create(entry)
+    .catch((error) => {
+      console.log('catch');
+      return error;
```

Comment:
Where is the error being returned to here and in the .catch block below? If return statements are going nowhere you might be able to save a couple lines of code